package.json

```diff
@@ -62,6 +62,7 @@
   "peerDependencies": {
     "react": "^16.x || ^17.x",
     "react-dom": "^16.x || ^17.x",
+
```

Comment:
I think you can remove the blank lines in this file. Looks like a result of resolving merge conflicts.